app/scripts/modules/amazon/src/loadBalancer/configure/application/ALBListeners.tsx

```diff
@@ -771,6 +771,14 @@ const Action = (props: {
       (props.oidcConfigs &&
         props.oidcConfigs.length > 0 &&
         (!clientId || props.oidcConfigs.find(c => c.clientId === clientId)));
+
+    const oidcOptions =
+      props.oidcConfigs && props.oidcConfigs.length ? (
```

Comment:
might as well use some es2020 sugar, (and remove the || [] guard):
```suggestion
      props.oidcConfigs?.length ? (
              props.oidcConfigs.map(config => <option key={config.clientId}>{config.clientId}</option>)
```